NiL.JS/ExportTable.cs

```diff
@@ -28,7 +28,27 @@ internal set
                 _items[key] = value;
             }
         }
+        public void DefineConstructor(Type type, string name)
+        {
+            var ctor = GlobalContext.DefaultGlobalContext.GetConstructor(type);
+            _items.Add(name, ctor);
+            ctor._attributes |= JSValueAttributesInternal.DoNotEnumerate;
+        }
 
+        public JSValue DefineVariable(string name, bool deletable)
+        {
+            var defineVariable = GlobalContext.DefaultGlobalContext.DefineVariable(name, deletable);
+            _items.Add(name,defineVariable);
+            return defineVariable;
+        }
+        public JSValue DefineConstant(string name, JSValue value, bool deletable = false)
+        {
+            var v = DefineVariable(name, deletable);
+            v.Assign(value);
+            v._attributes |= JSValueAttributesInternal.ReadOnly;
+            return v;
+        }
+        
```

Comment:
What difference between DefaultGlobalContext and CurrentGlobalContext?

Reply:
> At least, `GlobalContext.DefaultGlobalContext` should be replaced with `GlobalContext.CurrentGlobalContext`. But better way is to remove direct link to context at all. As i realise few days ago, current implementation of Context.Current* is not safe. Let's remove this methods and just make setter of `this` public.

Hey
Maybe dont remove this methods because is very usable (get constructor or define variable), but mark as not recommend for use?